README.md

```diff
@@ -90,17 +90,14 @@ Kops also has time set aside every other week to offer help and guidance to the
 community. Kops maintainers have agreed to set aside time specifically dedicated
 to working with newcomers, helping with PRs, and discussing new features.
 
-We recommend letting us know **beforehand** if you plan on attending so we can
-have time to prepare for the call.
+Please feel free to reach out **beforehand** if you plan on attending!
 
-| Maintainer   | Schedule      |  URL |
```

Comment:
Why are we nuking the maintainers here? Just curious. I think it is important to let people know who they can expect to see on the call, also if they need to ping someone before/after